packages/client/src/containers/RetroPage/RetroPage.jsx

```diff
@@ -0,0 +1,119 @@
+import React, { useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { apiURL } from "../../apiURL";
+import PropTypes from 'prop-types'; 
+import './RetroPage.css';
+
+
+export default function RetroPage({ retroCode, setRetroCode }) {
```

Comment:
I don't see you passing these two props in the `App.js`

If you want to navigate to a specific `RetroPage`, I don't think this is the way to go.
Check your **meal sharing project** you most probably solved the same problem there.